test/web-platform-tests/to-run.yaml

```diff
@@ -126,7 +126,7 @@ DIR: dom/nodes
 Comment-constructor.html: [fail, Unknown]
 DOMImplementation-createDocument.html: [fail, Unknown]
 Document-URL.html: [fail, Unknown]
-Document-characterSet-normalization.html: [fail, Some encodings are not supported - see the whatwg-encoding module]
+Document-characterSet-normalization.html: [timeout, Some encodings are not supported - see the whatwg-encoding module]
```

Comment:
Why this change?

Reply:
With v12, I saw this test take various times between 10000ms to 60000ms until it failed.

Reply:
Interestingly it only seems to occur when all tests up to that point have been run. If just that test or only the tests from that directory run in isolation, I don't see the same long delay.